internal/controller/suite_test.go

```diff
@@ -78,6 +79,9 @@ func TestControllers(t *testing.T) {
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
+	//cfg, _, _, _, err := prepareK8SEnvironment()
+	//Expect(err).NotTo(HaveOccurred())
```

Comment:
I was trying to use the same common function here for both the new unit test and the previous unit tests, but for some reason it wasn't working here. I decided not to dwell on that since we plan on replacing the ginkgo struct-level unit tests with function-level unit tests.

Reply:
do you mind creating an issue for tracking this? Thanks!

Reply:
okay, I'm partly thinking it will be de-prioritized in favor of removing the test at some point, but in case we don't end up removing it, the issue can be there